src/libs/events.js

```diff
@@ -45,11 +46,16 @@ export const extractCrossWorkspaceDetails = (fromWorkspace, toWorkspace) => {
 }
 
 export const PageViewReporter = () => {
-  const { name } = useRoute()
+  const { name, params } = useRoute()
 
   useEffect(() => {
-    Ajax().Metrics.captureEvent(`${eventsList.pageView}:${name}`)
-  }, [name])
+    const isWorkspace = /^#workspaces\/.+\/.+/.test(window.location.hash)
+
+    Ajax().Metrics.captureEvent(
+      `${eventsList.pageView}:${name}`,
+      isWorkspace ? extractWorkspaceDetails(params) : undefined
+    )
+  }, [name]) // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
I think `params` should be included here. Since `name` is just something like `workspace-dashboard`, not including `params` would mean that we might miss events (which may actually be a bug today!).